### ISSUE TITLE
So this wont connect to just rethinkdb correct.

### ISSUE THREAD

**Issue by countrdd** [mariolenis/angular-rethinkdb#2]

**Comment by mariolenis** [comment]
Correct, unfortunately connection to rethinkDB can't be made using websockets (yet); also I think it would generate a lot of security problems. Think this solution more like a DaaS, like firebase.

I made this backend solution [https://github.com/mariolenis/rethinkdb-daas/blob/master/_src/daas/db.ts](https://github.com/mariolenis/rethinkdb-daas/blob/master/_src/daas/db.ts) it's not perfect, but it's a good approach.

There's also a project called Horizon for RethinkDB but i'm not cool with the target of the solution.

**Comment by countrdd** [comment]
Target of the solution?
